simple.js

```diff
@@ -1381,19 +1378,22 @@ define(function(require, exports, module) {
                 // set Python's version
                 settings.set("project/python/@version", "python3");
 
-                // set pylint's flags
-                settings.set(
-                    "project/python/@pylintFlags",
-                    "-d all -e E -e F --generated-members=app.jinja_env.* --ignored-classes=SQLAlchemy --load-plugins pylint50"
-                );
-
                 // set PYTHONPATH
                 settings.set(
                     "project/python/@path",
                     "/home/ubuntu/.local/lib/python3.4/site-packages" +
                     ":/usr/lib/python3/dist-packages" +
                     ":/usr/local/lib/python3.4/dist-packages"
                 );
+
+                // set pylint's flags
+                settings.set(
+                    "project/python/@pylintFlags",
+                    "-d all -e E -e F --generated-members=app.jinja_env.* --ignored-classes=SQL,SQLAlchemy --load-plugins pylint50"
```

Comment:
A `pylint` plugin which informs `pylint` that `cs50` has a member called `SQL`, and therefore stopping it from complaining about `SQL` not being a member of `cs50`. Source code is here: https://github.com/cs50/ide50/blob/develop/files/home/ubuntu/.local/lib/python3.4/site-packages/pylint50.py Documentation is here: https://docs.pylint.org/en/1.6.0/plugins.html, if interested.

If you're wondering why some/all of the other flags are not in that plugin too, that's because:

1. They exist, so they don't need a plugin to implement the behavior they induce. This is unlike ignoring `no-member` errors for particular members (e.g., `SQL` in this case). In other words, there's no flag to ignore specific `no-member` errors, while there's a flag for ignoring whole classes for example.

2. Specifying a command-line flag is much simpler than implementing the same behavior in a plugin (if actually possible?)

What might be a good place for all of these flags though is `~/.c9/.pylintrc`, but I think command-lines flags have higher priority, and we probably have to set `PYTHONPATH` for the Python language handler plugin there too, so it might be better to keep them in the same place since they're related to one plugin.